pkgs/development/ocaml-modules/digestif/default.nix

```diff
@@ -1,27 +1,23 @@
-{ lib, fetchurl, fetchpatch, buildDunePackage
+{ lib, ocaml, fetchurl, buildDunePackage
 , bigarray-compat, eqaf, stdlib-shims
 , alcotest
 }:
 
 buildDunePackage rec {
   pname = "digestif";
-  version = "0.8.0";
+  version = "0.9.0";
+
+  useDune2 = true;
 
   src = fetchurl {
     url = "https://github.com/mirage/digestif/releases/download/v${version}/digestif-v${version}.tbz";
-    sha256 = "09g4zngqiw97cljv8ds4m063wcxz3y7c7vzaprsbpjzi0ja5jdcy";
+    sha256 = "0vk9prgjp46xs8qizq7szkj6mqjj2ymncs2016bc8zswcdc1a3q4";
   };
 
-  # Fix tests with alcotest ≥ 1
-  patches = [ (fetchpatch {
-    url = "https://github.com/mirage/digestif/commit/b65d996c692d75da0a81323253429e07d14b72b6.patch";
-    sha256 = "0sf7qglcp19dhs65pwrrc7d9v57icf18jsrhpmvwskx8b4dchfiv";
-  })];
-
-  buildInputs = lib.optional doCheck alcotest;
   propagatedBuildInputs = [ bigarray-compat eqaf stdlib-shims ];
 
-  doCheck = true;
+  checkInputs = [ alcotest ];
```

Comment:
`digestif.opam` lists `bos`, `astring`, `fpath`, `rresult` as test dependencies and `findlib` as build dependency. Maybe sensible to add the explicitly instead of relying on propagation?

Reply:
Maybe the `digestif.opam` is out of date?

Reply:
Seems to be necessary for <https://github.com/mirage/digestif/blob/f97216b8f3a501c8f0197be338ce85145fb46443/test/test_runes.ml>. Seems like this test is invoked separately which we could (should?) do as well.

Reply:
Well spotted. I’ve thus added a `postCheck` attribute to run that lone test explicitly.